src/test/ui/rfc-3086-metavar-expr/syntax-errors.stderr

```diff
@@ -0,0 +1,135 @@
+note: Try removing `$`
+  --> $DIR/syntax-errors.rs:16:39
+   |
+LL |     ( $( $i:ident ),* ) => { ${ count($i) } };
+   |                                       ^
+
+note: meta-variable expressions do not expect arbitrary prefixes
+  --> $DIR/syntax-errors.rs:16:33
+   |
+LL |     ( $( $i:ident ),* ) => { ${ count($i) } };
+   |                                 ^^^^^
+
+error: a `ident` element was expected
+  --> $DIR/syntax-errors.rs:16:33
+   |
+LL |     ( $( $i:ident ),* ) => { ${ count($i) } };
+   |                                 ^^^^^
```

Comment:
I think you want to attach the note to the error so that it appears after the error in the output.

Reply:
But this note is exactly what I was thinking, and it is very helpful!

Reply:
> I think you want to attach the note to the error so that it appears after the error in the output.

Hope it is good now